toolbox/tabs/containers/ResizableContainerPanel.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import {hoistComponent} from 'hoist/core/index';
+import {hbox, vbox, box, h3} from 'hoist/layout/index';
+import {resizable} from 'hoist/cmp/resizable';
+import {wrapperPanel} from '../impl/WrapperPanel';
+
+@hoistComponent()
+export class ResizableContainerPanel extends Component {
```

Comment:
This is a nice example of the functionality.  Depending on which direction we take with the back-end server, we could also use this to demo the built-in preference support, which is a nice feature and helps to tell the overall preference story.